core/lib/protobuf_config/src/proto/config/da_dispatcher.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+package zksync.config.da_dispatcher;
+
+import "zksync/config/object_store.proto";
+
+message DataAvailabilityDispatcher {
+  optional uint32 polling_interval = 1;
```

Comment:
Should be named `polling_interval_ms`; compare with other config fields with a unit of measurement.